features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/ServiceCatalog/Listing/Delete.jsx

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { useState } from 'react';
+import { FormattedMessage, injectIntl, useIntl } from 'react-intl';
+import PropTypes from 'prop-types';
+import Button from '@material-ui/core/Button';
+import Icon from '@material-ui/core/Icon';
+import Alert from 'AppComponents/Shared/Alert';
+import ConfirmDialog from 'AppComponents/Shared/ConfirmDialog';
+import ServiceCatalog from 'AppData/ServiceCatalog';
+
+/**
+* Service Catalog service delete
+* @param {any} props Props for delete function.
+* @returns {any} Returns the rendered UI for service delete.
+*/
+function Delete(props) {
+    const intl = useIntl();
```

Comment:
With this `useIntl` Usage of `injectIntl` is redundant and hasn't used the props passed from this HOC(Higher-order component) also.So shall we remove the `injectIntl` thinngs

Reply:
Fixed with https://github.com/wso2/carbon-apimgt/pull/9407/commits/a9d9f33a89aa8d840fac45a19758fa7270787f7c